simSPI/linear_simulator/modules.py

```diff
@@ -0,0 +1,397 @@
+"""contains simulator and its components classes."""
```

Comment:
Replace with what you wrote below: "Module to generate data using a linear forward model."